### DISCUSSION TITLE
Note effects + Beat effects on the same note?

### DISCUSSION THREAD

**Discussion by BlairBeverly** [CoderLine/alphaTab#1436] [Q&A] [answered]
Thanks for alphaTab! When using piano tuning, is there a way with alphatex to create a dotted quarter note that ties into another dotted quarter note? It seems like it's not possible to combine note effects and beat effects together on the same note, but I might be missing something. I read through [this documentation on tied notes](https://alphatab.net/docs/alphatex/note-effects#tied-notes), but couldn't find anything for my use case. The following approaches I tried did not work:

G4.4{d} G4.4{d}{-}
G4.4{d} G4.4{d}{t}
G4.4{d} G4.4{-}{d}
G4.4{d} G4.4{t}{d}

Any help would be appreciated. Thanks!

**Comment by Danielku15** [comment, marked as answer]
Variants that work: 

- `G4.4{d} -.4{d}`
- `G4.4{d} G4{t}.4{d}`
- `G4.4{d} G4{-}.4{d}`

![image](https://github.com/CoderLine/alphaTab/assets/674916/914286ab-b0cb-4045-9c60-ab9728cb2abb)

Note effects have to be put directly after the note and beat effects after the beat. For piano tuning notes the format is typically `note.beatduration` and with effects its `note{note effects}.beatduration{beat effects}`

Hope that helps. 😉

**Reply by BlairBeverly** [reply]
Thanks -worked!! Appreciate it.
